sources/packages/backend/apps/db-migrations/src/sql/Queue/Add-student-application-notifications-queue.sql

```diff
@@ -0,0 +1,18 @@
+INSERT INTO
```

Comment:
Formatting is not right, please check if all the plugins for formatter is in place.
https://github.com/bcgov/SIMS/wiki/Development---Standards,-Decisions,-and-Troubleshooting#plugins-to-be-installed

Reply:
The mentioned extension is the [SQL Formatter](https://marketplace.visualstudio.com/items?itemName=adpyke.vscode-sql-formatter) mentioned in the link shared by @guru-aot.
The format change is minor for this SQL, but not having it installed will be a problem for more complex queries. Please ensure that it is installed and formatting the code while saving.